.envrc

```diff
@@ -24,7 +24,7 @@ export FISSILE_DOCKER_REGISTRY="${FISSILE_DOCKER_REGISTRY:+${FISSILE_DOCKER_REGI
 export FISSILE_DOCKER_ORGANIZATION=${FISSILE_DOCKER_ORGANIZATION:-'splatform'}
 
 FISSILE_RELEASE=""
-FISSILE_RELEASE="${FISSILE_RELEASE},${PWD}/src/scf-helper-release"
+#FISSILE_RELEASE="${FISSILE_RELEASE},${PWD}/src/scf-helper-release"
```

Comment:
The line should be deleted and not commented out. We have version control to get the history of changes.

Reply:
If I remove this line then the line above and the line below are no longer needed. Then `bin/diff-releases.sh` is not needed as well :). Will perform a more brutal cleanup in a new commit.